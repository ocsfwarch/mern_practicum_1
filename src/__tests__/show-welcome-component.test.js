import React from 'react';
import ShowWelcome from '../components/show-welcome.component.js';
import renderer from 'react-test-renderer';

it('Renders a snapshot', () => {
  const tree = renderer.create(<ShowWelcome/>).toJSON();
  expect(tree).toMatchSnapshot();
});
