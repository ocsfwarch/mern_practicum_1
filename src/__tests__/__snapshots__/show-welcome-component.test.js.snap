// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a snapshot 1`] = `
<div>
  <h3>
    MERN Stack Web Dev Practicum
  </h3>
  <p>
    This application will allow users the ability to upload/record speech/conversation of up to 60 minutes, translates the audio and sends a text transcript to th user's email upon completion.
    <br />
    <strong>
      Upload Audio
    </strong>
     - Select this menu item to upload an existing audio file for translation.
    <br />
    <strong>
      Create Audio
    </strong>
     - Select this menu item to record a live audio file for translation.
    <br />
    <strong>
      Transcript List
    </strong>
     - Select this menu item to search old transcripts.
  </p>
</div>
`;
